randovania/gui/main_window.py

```diff
@@ -190,18 +206,56 @@ def __init__(
             logo.clicked.connect(partial(self._select_game, game))
             logo.setVisible(game.data.development_state.can_view())
 
-            def highlight_logo(label: ClickableLabel, active: bool):
-                if active:
-                    label.new_effect = QtWidgets.QGraphicsColorizeEffect()
-                    label.new_effect.setStrength(0.5)
-                    label.setGraphicsEffect(label.new_effect)
-                else:
-                    label.setGraphicsEffect(None)
-
-            logo.entered.connect(partial(highlight_logo, logo, True))
-            logo.left.connect(partial(highlight_logo, logo, False))
-            self.play_flow_layout.addWidget(logo)
-            self._play_game_logos[game] = logo
+            on_hover_effect = QtWidgets.QGraphicsColorizeEffect()
+            on_hover_effect.setStrength(0.5)
+            logo.setGraphicsEffect(on_hover_effect)
+            on_hover_effect.setEnabled(False)
+
+            if game.data.defaults_available_in_game_sessions:
+                multi_banner = QtWidgets.QLabel(pack_tile)
+                multi_banner.setPixmap(QtGui.QPixmap(os.fspath(banner_img_path)))
+                multi_banner.setScaledContents(True)
+                multi_banner.setFixedSize(bannerSize, bannerSize)
+                multi_banner.setVisible(game.data.defaults_available_in_game_sessions)
+
+                color_effect = QtWidgets.QGraphicsColorizeEffect()
+                color_effect.setStrength(0.5)
+                color_effect.setColor(QtGui.QColor(70, 200, 80))
+                multi_banner.setGraphicsEffect(color_effect)
+                multi_banner.move(0, 2)
+
+                multi_icon = QtWidgets.QLabel(pack_tile)
+                multi_icon.setPixmap(QtGui.QPixmap(os.fspath(multiworld_img_path)))
+                multi_icon.setScaledContents(True)
+                multi_icon.setFixedSize(int(bannerSize / 2), int(bannerSize / 2))
+                multi_icon.setVisible(game.data.defaults_available_in_game_sessions)
+                multi_icon.move(int(bannerSize / 4), int(2 + bannerSize / 4))
+                if multi_icon.isVisible:
```

Comment:
As already have been mentioned, this is not a bool value but a function returning a bool.
Also the check doesn't make sense at all?  The whole block is in a if case from line 214 where it checks for `game.data.defaults_available_in_game_sessions` which is also used to two lines above this one `multi_icon.setVisible(game.data.defaults_available_in_game_sessions)`.
In this code block, it should be always visible and truthy.

Which means to me: Line 231 is just `multi_icon.setVisible(True)` and the if condition in Line 233 is not needed.